metadata.rb

```diff
@@ -10,7 +10,6 @@
 
 depends          'apt'
 depends          'certificate'
-depends          'cron', '~> 7.0.20'
```

Comment:
You still need to keep this, just remove the version constraint please.

Reply:
Ah yes I added this back yesterday briefly after removing the whole line by accident.